fabric-tool-attribute-v1/src/main/java/net/fabricmc/fabric/api/tool/attribute/v1/MiningConstants.java

```diff
@@ -0,0 +1,22 @@
+package net.fabricmc.fabric.api.tool.attribute.v1;
+
+/**
+ * Constants for all the mining levels and speed multipliers of vanilla materials.
+ */
+public final class MiningConstants {
+	private MiningConstants() { }
+
+	public static final int WOOD_MINING_LEVEL = 0;
+	public static final int STONE_MINING_LEVEL = 1;
+	public static final int IRON_MINING_LEVEL = 2;
+	public static final int DIAMOND_MINING_LEVEL = 3;
+	public static final int NETHERITE_MINING_LEVEL = 3;
+	public static final int GOLD_MINING_LEVEL = 0;
+
+	public static final float WOOD_MINING_SPEED_MULTIPLIER = 2;
+	public static final float STONE_MINING_SPEED_MULTIPLIER = 4;
+	public static final float IRON_MINING_SPEED_MULTIPLIER = 6;
+	public static final float DIAMOND_MINING_SPEED_MULTIPLIER = 8;
+	public static final float NETHERITE_MINING_SPEED_MULTIPLIER = 9;
+	public static final float GOLD_MINING_SPEED_MULTIPLIER = 12;
```

Comment:
Is there a way to refer to a concrete value in the game rather than constants mojang could change at any time during snapshots?

Reply:
This would rather be impl specific. I think currently holding these in constants shouldn't be too bad, but I don't know if the javac will optimize these field calls in mods so that they get replaced by senseless literal numbers.

Reply:
I wonder if we want these fields anyways, as this is just for reference, I feel like mod developers might want to just do `ToolMaterials.GOLD.<insert method to get mining speed>`

Reply:
I put em in because Player recommended em.

Reply:
If a mod changes the vanilla values with a mixin and something (maybe because of reasons), these will not be correct, I feel like we shouldn't add this just because this is a handy reference.